include/SDL3/SDL_main.h

```diff
@@ -140,20 +140,44 @@
  *  \endcode
  */
 
-#if defined(SDL_MAIN_NEEDED) || defined(SDL_MAIN_AVAILABLE)
+#if defined(SDL_MAIN_NEEDED) || defined(SDL_MAIN_AVAILABLE) || defined(SDL_MAIN_USE_CALLBACKS)
 #define main    SDL_main
 #endif
 
+#include <SDL3/SDL_events.h>
```

Comment:
Forward declare SDL_Event?
Not a big deal, it just seems like an odd place for a header.

Reply:
Fixed in a22747e3686647636bb8830c8cf6738e94dbab54